examples/tutorials/advanced/websockets-example-MNIST/Federated learning with websockets and federated averaging.ipynb

```diff
@@ -142,7 +168,109 @@
     "\n",
     "We instantiate two FederatedDataLoaders, one for the train and one for the test set of the MNIST dataset.\n",
     "\n",
-    "*If you run into BrokenPipe errors try to restart the notebook or try running [run_websocket_client.py](run_websocket_client.py) (Python script that contains the same code of this notebook)*"
```

Comment:
I think we can remove the comment concerning the BrokenPipe issue. Should be resolved by now (improved startup parameters of the websocket connection).

Reply:
this error happened to me 2 days ago when I was trying to run that box. When was the issue resolved?

Reply:
I don't really use the notebooks, so it seems that I confused it with another problem. If it persists, leave it in.